app/src/main/java/com/feedhenry/securenativeandroidtemplate/features/authentication/providers/KeycloakAuthenticateProviderImpl.java

```diff
@@ -55,61 +63,54 @@
     private AuthorizationService authService;
     private AuthorizationRequest authRequest;
     private AuthorizationServiceConfiguration serviceConfig;
-    private Callback authCallback;
-    private Callback logoutCallback;
     private AuthStateService authStateService;
+    private MobileCoreService mobileCoreService;;
+    public static int LOGIN_RESULT_CODE = 1;
 
     @Inject
     Context context;
 
     @Inject
-    public KeycloakAuthenticateProviderImpl(@NonNull Context context, AppConfiguration appConfiguration, AuthStateService authStateService) {
+    public KeycloakAuthenticateProviderImpl(@NonNull Context context, AppConfiguration appConfiguration, AuthStateService authStateService, MobileCoreService mobileCoreService) {
         this.context = context;
         this.authenticationConfiguration = appConfiguration.getAuthConfiguration();
         this.authStateService = authStateService;
+        this.mobileCoreService = mobileCoreService;
```

Comment:
should be able to use DI to inject the AuthService directly. Do we actually need the MobileCoreService here?